internal/armada/repository/job_test.go

```diff
@@ -189,59 +188,6 @@ func TestDeleteQueuedJob(t *testing.T) {
 	})
 }
 
-func TestDeleteJobShouldSetJobObjectToExpire(t *testing.T) {
```

Comment:
Do we rely on being able to return leases for deleted jobs?

Reply:
Not conceptually but possibly inadvertently - I can think of edge cases where it could happen - however as long as the server treats it as a noop it should be fine.

Reply:
Done